RecoPixelVertexing/PixelTriplets/python/pixelTripletHLTEDProducer_cff.py

```diff
@@ -0,0 +1,7 @@
+from RecoPixelVertexing.PixelTriplets.pixelTripletHLTEDProducer_cfi import *
+from Configuration.Eras.Modifier_trackingLowPU_cff import trackingLowPU
+from Configuration.Eras.Modifier_trackingPhase1PU70_cff import trackingPhase1PU70
+from Configuration.Eras.Modifier_trackingPhase2PU140_cff import trackingPhase2PU140
+trackingLowPU.toModify(pixelTripletHLTEDProducer, maxElement=100000)
+trackingPhase1PU70.toModify(pixelTripletHLTEDProducer, maxElement=0)
+trackingPhase2PU140.toModify(pixelTripletHLTEDProducer, maxElement=0)
```

Comment:
shouldn't this be applied to a clone from the fillDescriptions-generated pixelTripletHLTEDProducer_cfi?
I'd like to avoid a pattern where pixelTripletHLTEDProducer_cfi is loaded instead of cff and the era modifications are missed